src/ResourceManager/HDInsight/Commands.HDInsight/ManagementCommands/AddAzureHDInsightComponentVersionCommand.cs

```diff
@@ -47,12 +47,12 @@ public class AddAzureHDInsightComponentVersionCommand : HDInsightCmdletBase
 
         public override void ExecuteCmdlet()
         {
-            ConfirmAction("Adding Component Version", Name,
+            ConfirmAction("Adding Component Version", "AzureHDInsightConfig",
```

Comment:
I have attached a picture of the confirm action prompt for this cmdlet.
![componentversioncmdlet](https://cloud.githubusercontent.com/assets/14077958/19734609/052ba8a0-9b5d-11e6-9eee-58c65964bf69.JPG)